### PR TITLE
Header of Gas tank UI screen now fetched from Entity name and locale …

### PR DESCRIPTION
**Pull request by Laneron**
## About the PR
Changed displayed header name of UI window

## Why / Balance
https://github.com/space-wizards/space-station-14/issues/27242

## Technical details
Changed logic of displayed window header invoked by "Toggle UI" button from hardcoded "Gas Tank" to dependable from entity name

image

**Changelog**
- fix: Jetpack UI window now has correctly displayed header

### CODE REVIEW

File: Content.Client/UserInterface/Systems/Atmos/GasTank/GasTankWindow.cs [hunk 1]
```diff
@@ -15,7 +15,9 @@ namespace Content.Client.UserInterface.Systems.Atmos.GasTank
     public sealed class GasTankWindow
         : BaseWindow
     {
+
         private GasTankBoundUserInterface _owner;
+        private readonly IEntityManager _entityManager;
```
Comment (metalgearsloth) [inline Content.Client/UserInterface/Systems/Atmos/GasTank/GasTankWindow.cs:20]: This isn't even used outside of the ctor?

File: Content.Client/UserInterface/Systems/Atmos/GasTank/GasTankWindow.cs [hunk 2]
```diff
@@ -27,12 +29,13 @@ public sealed class GasTankWindow
...
 
-        public GasTankWindow(GasTankBoundUserInterface owner)
+        public GasTankWindow(GasTankBoundUserInterface owner, IEntityManager entityManager)
         {
             TextureButton btnClose;
             _resourceCache = IoCManager.Resolve<IResourceCache>();
             _owner = owner;
-            var rootContainer = new LayoutContainer {Name = "GasTankRoot"};
+            _entityManager = entityManager;
+            var rootContainer = new LayoutContainer { Name = "GasTankRoot" };
```
Comment (metalgearsloth) [inline Content.Client/UserInterface/Systems/Atmos/GasTank/GasTankWindow.cs:38]: Just use IoCManager.Instance and resolve both manually as the entmanager isn't used outside of here.

**Comment by metalgearsloth** [comment]
Screenshots please

**Comment by Laneron** [comment]
> Screenshots please

Screenshot attached

**Pull request closed without merging**